account_chatter/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Account Chatter",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo",
+    "category": "Accounting",
+    "website": "http://www.vauxoo.com",
+    "license": "LGPL-3",
+    "depends": [
+        "account",
+    ],
+    "demo": [
+
+    ],
+    "data": [
+        "security/security.xml",
+        "views/account_account.xml",
```

Comment:
Please add suffix `_views` to all views, this to conform guidelines.

```suggestion
        "views/account_account_views.xml",
```

BTW, I like that files already refers to the full model name.